rest-server/src/controllers/job.js

```diff
@@ -15,26 +15,24 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-
 // module dependencies
 const Job = require('../models/job');
 const logger = require('../config/logger');
 
-
 /**
  * Load job and append to req.
  */
 const load = (req, res, next, jobName) => {
   job = new Job(jobName, () => {
     if (job.jobStatus.state === 'JOB_NOT_FOUND' && req.method !== 'PUT') {
       logger.warn('load job %s error, could not find job', jobName);
-      return res.status(500).json({
-        error: job.state,
+      return res.status(404).json({
+        error: "JobNotFound",
         message: `could not find job ${jobName}`
```

Comment:
Suggest capitalize the first letter of each message.

Reply:
I didn't change your code here. Would you like me to change?

Reply:
And I suggest we do it in another PR. This PR is sufficiently large.

Reply:
yes, we can standardize error message as Bin suggested in another pr.